appveyor.yml

```diff
@@ -55,8 +55,3 @@ on_success:
   - IF %APPVEYOR_REPO_TAG%==true node-pre-gyp publish
 
 build: off
-
-branches:
-  only:
-    - master
```

Comment:
Correct me if I'm wrong, but doesn't the removal of this line cause every branch on the nodegit repo to be built / tested on appveyor?

Reply:
Yes, and in every fork with has activated AppVeyor.
Having the CI running only in pull requests in annoying.

Reply:
I don't think I am willing to approve this change. For instance, some maintainers, not just myself, will push their branches to the nodegit repo, and then make PRs against master. In those instances, we will be double building branches for no reason. It will also clutter up our appveyor / travis test results.

Is it too much to ask to create a pull request when you are looking to test against CI?

Reply:
At best we could split this and the travis change out, and come back to it in a separate PR where we can have more discussion about it.